api-tests/test_update_responses.py

```diff
@@ -0,0 +1,62 @@
+import requests
+from io import StringIO
+from lxml import etree
+
+
+def test_update_addon_responses():
+    addon_urls = [
+        'https://aus5.mozilla.org/update/3/SystemAddons/44.0/20160310153207/Darwin_x86_64-gcc3-u-i386-x86_64/en-GB/release/Darwin%2014.5.0/default/default/update.xml',
```

Comment:
Ah, duh. I don't have a strong preference on which route we take here.

Eventually, we may want to support querying with Flask's test client (like we do for other CI tests: https://github.com/mozilla/balrog/blob/c6b8b919362ace0d819018cd602bafb5d9bfbdd5/auslib/test/web/test_client.py#L617), but that's not something we need to block on. There will probably be other changes needed when we do CI integration.